drivers/mtd/gd5f.c

```diff
@@ -1,9 +1,7 @@
 /****************************************************************************
  * drivers/mtd/gd5f.c
- * Driver for GigaDevice SPI nand flash.
```

Comment:
this one can change to apache, since fishsemi(xiaomi) already sign CLA.

Reply:
solved